docs/index.md

```diff
@@ -67,7 +67,7 @@ Here is documentation on the [GIS to GEXF](Tools/GIStoGraph.md) tool. This tool
 
 ## Code of Conduct
 
-Our [code of conduct](../CODE_OF_CONDUCT.md)
+Our [code of conduct](CODE_OF_CONDUCT.md)
```

Comment:
I think you need to use a full URL to link to the file in the main ("live") GH repo.